pepper-apis/src/main/java/org/broadinstitute/ddp/route/PutFormAnswersRoute.java

```diff
@@ -181,8 +181,13 @@ public Object handle(Request request, Response response) {
                                 formActivitySettingDto.get().shouldSnapshotSubstitutionsOnSubmit());
                         if (formActivitySettingDto.get().shouldSnapshotAddressOnSubmit()) {
                             var address = addressService.snapshotAddress(handle, userGuid, operatorGuid, form.getInstanceId());
-                            LOG.info("Created snapshotted address with guid {} for user {} and activity instance {}",
-                                    address.getGuid(), userGuid, instanceGuid);
+                            if (address != null) {
+                                LOG.info("Default address is snapshotted with guid {}, for user {}, activity instance {}",
+                                        address.getGuid(), userGuid, instanceGuid);
+                            } else {
+                                LOG.error("Address snapshotting failed - default address is not found. User {}, activity instance {}",
```

Comment:
This generates error alert into prod alerts slack channel. If we want to log and move on may be LOG.warn

Reply:
But as I understand such situation (address not found) is rather critical - should it be detected as soon as possible? In case of WARN we can not detect it (most likely we just not notice it).